pypeit/metadata.py

```diff
@@ -1328,22 +1286,29 @@ def set_pypeit_cols(self, write_bkg_pairs=False):
         # Return
         return output_cols
 
-    def set_defaults(self):
+    def set_combination_groups(self):
         """
-        Set default values for comb_id and calib
-        columns of the fitstbl
+        Set combination groups.
+
+        If the 'comb_id' or 'bkg_id' columns do not exist, they're set to -1.
 
-        self.table is modified in place
+        If none of the 'comb_id' values are other then -1, the
+        combination groups are set to be unique for each standard and
+        science frame.
+
+        :attr:`table` is edited in place.
 
         """
-        # Set comb_id
+        if 'comb_id' not in self.keys():
```

Comment:
Again, just checking that these tags don't show up in the .pypeit file
unless desired.

Reply:
Yep.